lab2/syntax.in.txt

```diff
@@ -2,6 +2,7 @@ Syntax:
 
 program = "begin;" decllist ";" cmpdstmt "end;"
 decllist = declaration ";"| declaration ";" decllist
+...
 declaration = "declare" type IDENTIFIER
 type1 = "boolean" | "char" | "integer" | "real"
 arraydecl = "declare" type IDENTIFIER"["nr"]"
```

Comment:
if cmpdstmt = stmtlist, you might remove one of them and replace its occurences with the other one, for redundancy